### PR TITLE
[DE-416] Add unlayer editor

### PR DESCRIPTION
**Pull request by Cromeror**
Using two path to editor:
* */campaigns/idCampaing
* */editors/templates/idTemplate

[DE-416](https://makingsense.atlassian.net/browse/DE-416)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +33,7 @@
     "eclint-check": "eclint check",
     "prettier-fix": "prettier --write \"**/**\"",
     "prettier-check": "prettier --check \"**/**\"",
-    "start": "react-scripts --openssl-legacy-provider start",
+    "start": "react-scripts start",
```
Comment (andresmoschini) [inline package.json:36]: Yo tube que hacer este cambio, sino en mi máquina funcionaba mal. A vos te pasó al revés?

File: src/components/Editor.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef } from "react";
+import EmailEditor, { UnlayerOptions, User } from "react-email-editor";
+import mergeTags from "../external/merge.tags";
+
+export interface EditorProps {
+  design?: any;
```
Comment (andresmoschini) [inline src/components/Editor.tsx:6]: What do you think about using a better type than `any` here? `react-email-editor` should define it.

File: src/components/Editor.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+export const Editor = (props: EditorProps) => {
+  const { design } = props;
```
Comment (andresmoschini) [inline src/components/Editor.tsx:10]: What do you think about:

```suggestion
export const Editor = ({ design }: EditorProps) => {
```

File: src/components/Editor.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef } from "react";
+import EmailEditor, { UnlayerOptions, User } from "react-email-editor";
+import mergeTags from "../external/merge.tags";
+
+export interface EditorProps {
```
Comment (andresmoschini) [inline src/components/Editor.tsx:5]: Exporting this type is not necessary, right?

File: package.json (hunk 1 above)
Comment (Cromeror) [reply package.json:36]: Sip tuve que devolverlo, no lo pensaba agregar en el commit pero agregue el cambio sin querer. ¿Lo revierto?

File: .gitignore [hunk 5]
```diff
@@ -30,3 +30,11 @@ yarn-error.log*
...
 .DS_Store
 Thumbs.db
+
+#InteliJ
+.idea
+
+# Environment
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (andresmoschini) [inline .gitignore:40]: These are already defined: 

image

File: package.json (hunk 1 above)
Comment (andresmoschini) [reply package.json:36]: No, dejalo como lo tenés vos que es el default, en todo caso de mi lado lo configuro diferente.

File: .gitignore (hunk 5 above)
Comment (Cromeror) [reply .gitignore:40]: 👍

File: package.json (hunk 1 above)
Comment (Cromeror) [reply package.json:36]: Dale 👍

File: src/components/Editor.tsx (hunk 3 above)
Comment (Cromeror) [reply src/components/Editor.tsx:10]: I plan to reuse the editor component and delegate the specific functionality to parent components (campaing | template).

File: src/components/Editor.tsx (hunk 4 above)
Comment (Cromeror) [reply src/components/Editor.tsx:5]: Yes right, not now.

**Comment by maurocardinali** [comment]
🚀 💪

**Comment by andresmoschini** [comment]
🚀

**Pull request merged**